Cargo.toml

```diff
@@ -26,6 +26,9 @@ members = [
     "core-primitives/test",
     "core-primitives/time-utils",
     "core-primitives/types",
+    "parentchain/block-importer",
+    "parentchain/indirect-calls-executor",
+    "parentchain/parentchain-crate",
     "service",
```

Comment:
I am uncertain if parentchain should be a separate domain. It is always needed, so I think we could put it into core?

Reply:
The fact that it's always needed does not disqualify it from being in its own domain in my opinion. We should group components independent of whether they're mandatory or not. I could move the entire `parentchain` sub-folder to `core`? And maybe then move `light-client` also into that sub-folder?

Reply:
You are right about that.

Yes, seems sensible to me. I also like the idea of having the `light-client` in the `parentchain` folder.

We could even argue to put sidechain into core, but I am not yet sure if this is a good idea.

Reply:
Okay, I've moved `parentchain` into `core` now and moved `light-client` into `parentchain` 👍